my-react-app/src/components/preloader/Preloader.css

```
@@ -0,0 +1,68 @@
+.preloader__container {
+	position: fixed;
+	width: 100%;
+	height: 100%;
+	background: rgba(41, 41, 41, 0.6);
+	top: 0;
+	left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.lds-ellipsis {
+  display: inline-block;
+  position: relative;
+  width: 80px;
+  height: 80px;
+}
+.lds-ellipsis div {
+  position: absolute;
+  top: 33px;
+  width: 13px;
+  height: 13px;
+  border-radius: 50%;
+  background: #fff;
+  animation-timing-function: cubic-bezier(0, 1, 1, 0);
+}
+.lds-ellipsis div:nth-child(1) {
```

You should not use `:nth-child()` selector to match single element. When you need single element it is better to use `.class` selector.
```suggestion
.lds-ellipsis .scale-up-ellips {
```
https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child